generator/template/vue-sections/pages/Sections.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div id="app">
+    <Sections
+      :admin="admin"
+      :pageName="pageName"
+      :variations="[]"
+      @finishLoad="sectionsLoaded"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "App",
+  data() {
+    return {
+      admin: false,
+      pageName: "home"
+    };
+  },
+  methods: {
+    sectionsLoaded() {
+      this.admin = !!this.$cookies.get("sections-auth-token");
```

Comment:
Please add a comment above this line to explain what it does

Reply:
Updated